src/content/docs/en/recipes/bun.mdx

```diff
@@ -117,6 +117,36 @@ Finally, use the following command to run your built site.
 bun ./dist/server/entry.mjs
 ```
 
+## Testing in Bun
+
+Bun ships with a fast, built-in, Jest-compatible test runner through the `bun test` command. If you like to use that read [`bun test` documentation](https://bun.sh/docs/cli/test).
+
+However, you may want to use Cypress or Playwright for a modern approach to testing web apps.
```

Comment:
```suggestion
However, it is also possible to use Cypress or Playwright for a modern approach to testing web apps.
```
I feel like this slight adjustment introduces the sections below a little better. "You might want to use..." is kind of a flat, "so what?" statement on its own that feels like it needs a follow up, "If so, then..."  Just saying stuff is possible gets to the point a little quicker!